package-lock.json

```diff
@@ -37845,8 +37852,7 @@
 			"requires": {
 				"@babel/runtime": "^7.8.3",
 				"debug": "^4.1.1",
-				"qs": "^6.5.2",
-				"wpcom-xhr-request": "^1.1.2"
```

Comment:
How come no more `wpcom-xhr-request` in this PR?

Reply:
Probably because `package-lock.json` hadn't been updated.

This is an expected change, since this package was never really used (we use `wpcom-proxy-request` instead of this in Calypso). See #39142.

Reply:
Right, this seems to be part of a stale-ish lockfile. I left babel package movement out, but I expect this to be safe.